Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset

```diff
@@ -1144,9 +1144,9 @@ MonoBehaviour:
       axisType: 7
       inputType: 3
       inputAction:
-        id: 4
-        description: Pointer Pose
-        axisConstraint: 7
+        id: 0
+        description: None
```

Comment:
Looks like the action mappings were lost when the keycodes / axes were updated.

Reply:
Odd, Unity mustn't have saved the asset properly, GAH. Fixing